cmd/kubeless/route/route.go

```diff
@@ -36,3 +42,60 @@ func init() {
 
 	}
 }
+
+func getRouteDefinition(funcObj spec.Function, funcName, hostName, ns string, enableTLSAcme bool) (v1beta1.Ingress, error) {
+	port := intstr.FromInt(8080)
+	if len(funcObj.Spec.ServiceSpec.Ports) != 0 {
+		port = funcObj.Spec.ServiceSpec.Ports[0].TargetPort
+	}
+
+	if port.IntVal <= 0 || port.IntVal > 65535 {
```

Comment:
AFAIK this check is not needed, port is either `8080` or is obtained from a default `funcObj` that should be properly built